### PR TITLE
Bug wave2 734 rounding image bricks

### PR DESCRIPTION
**Pull request by TDRoden**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/scss/ucgy_gbl_cta_hero.scss.bak.scss [hunk 1]
```diff
@@ -0,0 +1,779 @@
+@import '../../../../../themes/ucgy_theme/resources/scss/core';
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/scss/ucgy_gbl_cta_hero.scss.bak.scss]: Is there a reason this file was renamed to .bak?

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_related_content/scss/ucgy_gbl_related_content.scss [hunk 2]
```diff
@@ -123,14 +119,14 @@
...
 
-    &.rounded-brick-above {
-      .chicklet:nth-child(1),
-      .chicklet:nth-child(2),
-      .chicklet:nth-child(3){ // Only the tiles touching
-        height: 300px + $mobileRoundedBrick;
-        padding-top: 15px + $mobileRoundedBrick;
-      }
-    }
+    // &.rounded-brick-above {
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_related_content/scss/ucgy_gbl_related_content.scss]: Is this code going to be removed or is there a reason it has all been commented out?

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/scss/ucgy_gbl_cta_hero.scss.bak.scss (hunk 1 above)
Comment (TDRoden) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_cta/scss/ucgy_gbl_cta_hero.scss.bak.scss]: I am storing a whole bunch of work that I did to SASSify the Hero CTA code but was unable to fully complete. My goal is to pick it up again and complete the final tweaks and then that file can be deleted.

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_related_content/scss/ucgy_gbl_related_content.scss (hunk 2 above)
Comment (TDRoden) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_related_content/scss/ucgy_gbl_related_content.scss]: Have removed the code and additionally updated the comments to be more clear.

**Pull request merged**